frappe/model/db_schema.py

```diff
@@ -490,10 +490,13 @@ def create_user(self, user, password, host=None):
 		if not host:
 			host = self.get_current_host()
 
-		if password:
-			self.db.sql("CREATE USER '%s'@'%s' IDENTIFIED BY '%s';" % (user[:16], host, password))
-		else:
-			self.db.sql("CREATE USER '%s'@'%s';" % (user[:16], host))
+		try:
```

Comment:
@rmehta dont know how they got in was not planned the only change is setting the limit to 1000 now they have increased the limit from 255 to 64k(shared accross all columns in table not sure if check should be included for that)